qubit-sdk/src/main/java/com/qubit/android/sdk/internal/placement/interactor/PlacementInteractorImpl.kt

```diff
@@ -76,4 +112,15 @@ internal class PlacementInteractorImpl(
       onSuccess(null)
```

Comment:
Is it intentional, that null placement is treated as success?
If it is success, shouldn't placementModel be stored in cache when it is empty, too?

Reply:
Yes, returned placement data can be empty which should be treated as success.
But you're right, it should be cached in that case.